pkg/status/status.go

```diff
@@ -61,6 +62,18 @@ func GetStatus() (map[string]interface{}, error) {
 
 	stats["logsStats"] = logs.GetStatus()
 
+	otlpIsEnabled := otlp.IsEnabled(config.Datadog)
+	var otlpCollectorStatus, otlpCollectorStatusErr string
+	if otlpIsEnabled {
+		otlpCollectorStatus, otlpCollectorStatusErr = otlp.GetCollectorStatus(common.OTLP)
+	} else {
+		otlpCollectorStatus, otlpCollectorStatusErr = "N/A", ""
```

Comment:
Maybe "Not running" is more explicit here? To me "N/A" reads like "I couldn't get this value" when we actually know that the pipeline is not running
```suggestion
		otlpCollectorStatus, otlpCollectorStatusErr = "Not running", ""
```

Reply:
Fixed with [78f8c3d](https://github.com/DataDog/datadog-agent/pull/11822/commits/78f8c3dae5ded723138cf735a10699b072e08242)